aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -236,6 +271,7 @@
   private String dynamoDbTableName;
 
   private boolean isS3ChecksumEnabled;
+  private boolean isLakeFormationEnabled;
```

Comment:
nit: can you rename this as `glueLakeFormationEnabled`, and move it up to the Glue variables section (L268)?